### PR TITLE
MH3 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/flare_of_fortitude.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Flare of Fortitude
+ManaCost:Instant
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/flare_of_fortitude.txt:2]: Needs a cost, not a type

File: forge-gui/res/cardsfolder/upcoming/kozilek_the_broken_reality.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+SVar:TrigManifest:DB$ Manifest | ValidTgts$ Player | TargetMin$ 0 | Amount$ 2 | TargetMax$ 2 | RememberManifested$ True | SubAbility$ DBDraw | TgtPrompt$ Select up to two target players | Choices$ Card.TargetedPlayerOwn | ChoiceZone$ Hand | DefinedPlayer$ Targeted
+SVar:DBDraw:DB$ Draw | NumCards$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Remembered$Amount
+S:Mode$ Continuous | Affected$ Creature.Colorless+YouCtrl+Other | AddPower$ 2 | AddToughness$ 2 | Description$ Other colorless creatures you control get +2/+2.
+DeckHints:Color$Colorless & Type$Construct|Eldrazi|Juggernaut|Scarecrow
+Oracle:When you cast this spell, up to two target players each manifest two cards from their hands. For each card manifested this way, you draw a card. (To manifest a card, put it onto the battlefield face down as a 2/2 creature. Turn it face up any time for its mana cost if it's a creature card.)\nOther colorless creatures you control get +3/+2.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kozilek_the_broken_reality.txt:12]: +3 or 2?

File: forge-gui/res/cardsfolder/upcoming/shadow_of_the_second_sun.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Shadow of the Second Sun
+ManaCost:4 U U
+Types:Enchantment Aura
+K:Enchant player
+A:SP$ Attach | ValidTgts$ Player
+T:Mode$ Phase | Phase$ Main2 | ValidPlayer$ Player.EnchantedBy | TriggerZones$ Battlefield | Execute$ TrigAddPhase | TriggerDescription$ At the beginning of your postcombat main phase, there is an additional beginning phase after this phase. (The beginning phase includes the untap, upkeep, and draw steps.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shadow_of_the_second_sun.txt:6]: wrong description

File: forge-gui/res/cardsfolder/upcoming/ugins_binding.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Ugin's Binding
+ManaCost:2 U
+Types:Instant
+K:Devoid
+A:SP$ ChangeZone | ValidTgts$ Permanent.nonLand+YouDontCtrl | TgtPrompt$ Select target nonland permanent you don't control | Origin$ Battlefield | Destination$ Hand | SpellDescription$ Return target nonland permanent you don't control to its owner's hand.
+T:Mode$ SpellCast | ValidCard$ Creature.Colorless+cmcGE7+YouCtrl | TriggerZones$ Graveyard | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever you cast a colorless spell with mana value 7 or greater, you may exile CARDNAME from your graveyard. When you do, return each nonland permanent you don't control to its owner's hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ugins_binding.txt:6]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/fanatic_of_rhonas.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Fanatic of Rhonas
+ManaCost:1 G
+Types:Creature — Snake Druid
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fanatic_of_rhonas.txt:3]: typo

**Pull request merged**